examples/reminders/lib/base/common_blocs/coordinator_bloc.dart

```diff
@@ -103,44 +101,37 @@ extension CoordinatingTasksX on CoordinatorBlocType {
                   reminderList,
                   operationCallback: operationCallback,
                 )
-                .map((managedList) =>
-                    ManagedListCounterOperation<ReminderModel>(
-                        managedList: managedList,
-                        counterOperation: CounterOperation.create)),
+                .map(
+                  (managedList) => ManagedListCounterOperation<ReminderModel>(
+                      managedList: managedList,
+                      counterOperation: CounterOperation.create),
+                ),
             states.onReminderDeleted
                 .whereSuccess()
                 .withLatestFromIdentifiableList(
                   reminderList,
-                  operationCallback: (Identifiable reminder,
-                          List<ReminderModel> list) async =>
-                      ManageOperation.remove,
+                  operationCallback:
+                      (Identifiable reminder, List<ReminderModel> list) async =>
+                          ManageOperation.remove,
                 )
-                .map((managedList) =>
+                .map(
+                  (managedList) => ManagedListCounterOperation<ReminderModel>(
+                      managedList: managedList,
+                      counterOperation: CounterOperation.delete),
+                ),
+            states.onReminderUpdated.whereSuccess().switchMap(
+              (event) {
+                var stream =
```

Comment:
This smells kinda fishy ... can't we `withLatestFromIdentifiableList` in the outer stream, something like

states.onReminderUpdated.whereSuccess().withLatestFromIdentifiableList....